homeassistant/components/youtube/entity.py

```diff
@@ -0,0 +1,38 @@
+"""Entity representing a YouTube account."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.const import ATTR_ID
+from homeassistant.helpers.device_registry import DeviceEntryType
+from homeassistant.helpers.entity import DeviceInfo, Entity, EntityDescription
+
+from .const import ATTR_TITLE, DOMAIN, MANUFACTURER
+from .coordinator import YouTubeDataUpdateCoordinator
+
+
+class YouTubeChannelEntity(Entity):
```

Comment:
This should probably inherit `CoordinatorEntity`. Otherwise the coordinator won't poll and the entity state won't update.

Reply:
I couldn't figure out why the async time changed didn't work like intended, I guess this was the reason